src/index.ts

```diff
@@ -105,30 +151,22 @@ function _hydrate(line: Content, node: any, shouldExpandOnMouseOver: boolean = f
     }
   } else {
     // otherwise, this is a leaf node
-    if (lineText.includes("@Q ")) {
```

Comment:
this got moved up into the helper function. Side effect here is that it doesn't just act on leaf nodes anymore